lectures/ar1_processes.md

```diff
@@ -56,7 +56,7 @@ The *AR(1) model* (autoregressive model of order 1) takes the form
 X_{t+1} = a X_t + b + c W_{t+1}
 ```
 
-where $a, b, c$ are scalar-valued parameters.
+where $a, b, c$ are scalar-valued parameters and the equation is a **stochastic recurrence relations**.
```

Comment:
@SylviaZhaooo the `**` bold font is reserved for definitions. I think this should be `*`